x/ecocredit/core/msg_bridge_test.go

```diff
@@ -153,34 +158,17 @@ func TestMsgBridge(t *testing.T) {
 				},
 				Target:    "polygon",
 				Recipient: addr1,
-				Contract:  contract,
-			},
-			expErr: true,
-		},
-		"invalid bridge target": {
-			src: MsgBridge{
-				Owner: addr1,
-				Credits: []*Credits{
-					{
-						BatchDenom: batchDenom,
-						Amount:     "10",
-					},
-				},
-				Target:    "polygon1",
-				Recipient: recipient,
-				Contract:  contract,
 			},
-			expErr: true,
+			expErr:    true,
+			expErrMsg: "not a valid ethereum address",
 		},
 	}
 
-	for msg, test := range tests {
-		t.Run(msg, func(t *testing.T) {
-			t.Parallel()
```

Comment:
FYI. This does not work as intended and instead runs the same test multiple times without running the others.